PlayTools/MysticRunes/PlayedApple.swift

```diff
@@ -0,0 +1,164 @@
+//
+//  PlayWeRuinedIt.swift
+//  PlayTools
+//
+//  Created by Venti on 16/01/2023.
+//
+
+import Foundation
+import Security
+
+// Implementation for PlayKeychain
+// World's Most Advanced Keychain Replacement Solution:tm:
+// This is a joke, don't take it seriously
+
+public class PlayKeychain: NSObject {
+    static let shared = PlayKeychain()
+
+    private static func getKeychainDirectory() -> URL? {
+        let bundleID = Bundle.main.infoDictionary?["CFBundleIdentifier"] as? String ?? ""
+        let keychainFolder = URL(fileURLWithPath: "/Users/\(NSUserName())/Library/Containers/io.playcover.PlayCover")
```

Comment:
``` swift
let libraryRoute = FileManager.default.urls(for: .libraryDirectory, in: .userDomainMask).first
let keychainFolder = libraryRoute?.appendingPathComponent("Containers/io.playcover.PlayCover/PlayChain").appendingPathComponent("PlayChain").appendingPathComponent(bundleID)
```

This is one more line of code but should reduce hardcode. It's totally just syntax sugar.

Reply:
Done and done!

Reply:
Uh oh. Seems like that will just break the centralized keychain storage. I forgot that the FileManager here will be from the process's perspective, so the one above will actually resolve into `/Users/<User>/Library/Containers/<RunningAppContainer>/Data/Library/Containers/io.playcover.PlayCover/PlayChain/<RunningAppBundleID>/`
Not good.

Reply:
Change it to `.userDirectory` and change the second line to 
```
libraryRoute?.appendingPathComponent("Library")
                       .appendingPathComponent("Containers")
                       .appendingPathComponent("io.playcover.PlayCover")
```